packages/create-remix/templates/fly-stack/app/db.server.ts

```diff
@@ -0,0 +1,19 @@
+import { PrismaClient } from "@prisma/client";
+
+let prisma: PrismaClient;
+
+declare global {
+  var __db__: PrismaClient;
+}
+
```

Comment:
Could we add comments in here to explain why this is so complicated? (similar to what we do here: https://remix.run/docs/en/v1/tutorials/jokes#connect-to-the-database)